release_notes/v0.8.md

```diff
@@ -0,0 +1,174 @@
+# OPEA Highlights
+- In this release, OPEA supports large vision models and provides an initial Agent microservice. It also integrates more components, such as vLLM on Ray, giving users more choices for different AI functionalities. We have expanded the application range, enabling ChatQnA on AIPC and validating it on AWS. Additionally, more examples, such as FAQGen, are coming. Deployment is important, Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created.
```

Comment:
Should we add that we tested OPEA pipeline works on Nvidia?

Reply:
good idea